README.md

```
@@ -1,41 +1,48 @@
-If you reach out to me by email, please use a subject that starts with `[Github]` or I will treat the message as spam.
-I do like hearing from people who genuinely want to learn or have shared interests.
-I've been a part of a number of small companies and startups at all stages of their life and I'd love to hear about your side project or offer feedback where I'm able.
+If you reach out to me by email, please use a subject that starts with `[Github]` or I will treat
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)